editor/js/ui/projects.js

```diff
@@ -607,7 +670,9 @@ RED.projects = (function() {
                 row.click(function(evt) {
                     $('.projects-dialog-project-list-entry').removeClass('selected');
                     header.addClass('selected');
-                    $("#projects-dialog-open").prop('disabled',false).removeClass('disabled ui-button-disabled ui-state-disabled');
+                    if (options.clickHook) {
```

Comment:
We do not need `options.clickHook` - there is already `options.select` which does the same thing - see line 677 below.